packages/suite/src/services/suite/invityAPI.ts

```diff
@@ -474,7 +414,7 @@ class InvityAPI {
         try {
             const response: CoinmarketWatchTradeResponseMapProps[T] = await this.request(
                 tradesData.url.replace('{{counter}}', counter.toString()),
-                tradeData,
+                tradeData as any,
```

Comment:
We should not need `any` here (or anywhere else 😉)
```suggestion
                trade,
```